store/tikv/oracle/oracle.go

```diff
@@ -13,11 +13,14 @@
 
 package oracle
 
-import "time"
+import (
+	"golang.org/x/net/context"
+	"time"
+)
```

Comment:
Use alias for package name, like this:

```
goctx "golang.org/x/net/context"
```

because there is already "github.com/pingcap/tidb/context" in the repo,
so we choose that alias TiDB project.